structurizr-plantuml/src/com/structurizr/io/plantuml/PlantUMLWriter.java

```diff
@@ -42,10 +43,22 @@ public PlantUMLWriter() {
         addSkinParam("noteBorderColor", "#707070");
     }
 
+    public void addHeaderLine(final String line) {
```

Comment:
Rather than addHeaderLine(), what about having a specific method for each of the various PlantUML preprocessing commands; addInclude(), addIncludeUrl(), etc? We could then do some validation on URLs, for example, which will make the PlantUML writer more robust.

Reply:
Seems reasonable to do for !include & !includeurl although obviously we're imposing where in the file these get placed. That'll certainly get me the value I need.

Not sure how much validation we can do for other preprocessing statements, or whether they have use cases within PlantUMLWriter. I'll probably leave those as an exercise for the future.

Reply:
Added .addIncludeFile(..) and .addIncludeURL(..) methods as discussed.

Also modified diagram header to include the diagram key so that individual diagrams can be included from a file/url containing multiple. i.e. diagrams start with `@startuml(id=contextViewKey` and that can be included elsewhere via `!include filename.plantuml!contextViewKey` (Who knew that was an option!?)